stable/prisma/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: prisma
-version: 1.1.0
-appVersion: 1.20
+version: 1.1.1
```

Comment:
Can you make this a minor bump instead of a patch?

Reply:
of course

Reply:
@giacomoguiulfo  done!